stratum/hal/lib/common/common.proto

```diff
@@ -385,11 +385,11 @@ message OpticalPort {
   uint32 module_location = 7; // required
   // The unique netif location that represents the optical port on the module.
   uint32 netif_location = 8; // required
-  // Optical channel frequency in MHz.
+  // Optical channel frequency in Hz.
```

Comment:
Note: we still need to convert it to MHz.
https://github.com/openconfig/public/blob/master/release/models/optical-transport/openconfig-terminal-device.yang#L1205

But need to add this log some where

Reply:
We can have this logic in the pull functor (right before we send the data to controller)

Reply:
I saw that too. There and in the Frequency changed Event?

Reply:
Need to add logic to on pull, on change, and on set handler